Team 3749 2023/src/main/java/frc/robot/subsystems/Base.java

```diff
@@ -25,7 +25,7 @@ public Base() {
         neo.setInverted(true);
```

Comment:
Base.java and BaseCommand.java can be deleted

Reply:
we've kept them for now in case we need them later
and besides, other than being an extra file, it's not doing any harm.